packages/tools/generator-widget/CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 # Changelog
-All notable changes to this widget will be documented in this file.
+
+All notable changes to this tool will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## 9.0.1 - 2021-04-26
+
+### Changed
+
+- Explain users in the README of generated widget project how to install dependencies with NPM v7.x.x.
+
 ### Fixed
-- Fixes failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
-- Explains users in the README of generated widget project how to install dependencies with NPM v7.x.x.
-- Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
 
-## [major.minor.patch] - YYYY-MM-DD
+- Fix failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
+- Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
```

Comment:
Not entirely whether we need to be so specific about _why_ we made the changes. It doesn't hurt, but not sure about the added value of it. No need to necessarily change it tho.

Reply:
I can understand your doubts. Here I think it adds value, because it clarifies this isn't a breaking change.